wd_core.au3

```diff
@@ -726,9 +726,9 @@ EndFunc   ;==>_WD_FindElement
 ; ===============================================================================================================================
 Func _WD_ElementAction($sSession, $sElement, $sCommand, $sOption = Default)
 	Local Const $sFuncName = "_WD_ElementAction"
-	; because $sOption can contain sensitive data, do not pass untill $_WD_DEBUG_Full is used (refers to the case when VALUE will be set)
+	; because $sOption can contain sensitive data, mask value unless $_WD_DEBUG_Full is used (refers to the case when VALUE will be set)
 	Local $bParameters_Option = ($_WD_DEBUG = $_WD_DEBUG_Full Or Not ($sCommand = 'VALUE' And $sOption))
-	Local Const $sParameters = "Parameters:   Command=" & $sCommand & (($bParameters_Option) ? ("   Option=" & $sOption) : (""))
+	Local Const $sParameters = "Parameters:   Command=" & $sCommand & "   Option=" & (($bParameters_Option Or Not $sOption) ? ($sOption) : ("<masked>"))
```

Comment:
Here is not support for $sOption =Default

Reply:
I mean that 
```autoit
($bParameters_Option Or Not $sOption)
```
shoud be changed to :
```autoit
($bParameters_Option Or Not $sOption Or $sOption = Default)
```

but we can move this checkig to:

```autoit
Local $bParameters_Option = ($_WD_DEBUG = $_WD_DEBUG_Full Or Not ($sCommand = 'VALUE' And $sOption)  Or Not $sOption Or $sOption = Default)
```

Reply:
Thus we can do it even easier:
```autoit
Local $bParameters_Option = ($_WD_DEBUG = $_WD_DEBUG_Full Or Not $sOption Or $sOption = Default)
```

Reply:
I believe it worked correctly when I tested it earlier. Please double check before submitting revision.

Reply:
Ok.
I rechecked it all is fine.